scripts/codes/business-investment-instrument-asset.json

```diff
@@ -0,0 +1,13 @@
+{"_id":"9c036d23-373e-445b-8385-dd844a09ffdb","code":"N1132G","label":"ICT equipment (gross)","links":{"code_list":{"id":"business-investment-instrument-asset","href":"http://localhost:22400/code-lists/business-investment-instrument-asset"},"self":{"id":"N1132G","href":"http://localhost:22400/code-lists/business-investment-instrument-asset/codes/N1132G"}}}
```

Comment:
I suppose its down to how the business area defines their data, but `(gross)` on the end of every single value seems redundant, if it is within our power to address?

Reply:
same point as below (I started at the bottom for some reason). Basically, we could change it, but as they're globally defined resources I'd be wary of doing so.

example: https://registry.sdmx.org/FusionRegistry/ws/rest/codelist/ESTAT/CL_INSTR_ASSET/1.1/?detail=full&references=none&version=2.0